package.json

```diff
@@ -16,13 +16,17 @@
   "readmeFilename": "README.md",
   "//": "Callstats.io does not work with recent versions of jsSHA (2.0.1 in particular)",
   "dependencies": {
-    "@atlaskit/button": "1.0.3",
-    "@atlaskit/button-group": "1.0.0",
+    "@atlaskit/avatar": "4.0.5",
+    "@atlaskit/button": "3.0.0",
+    "@atlaskit/button-group": "1.1.3",
     "@atlaskit/dropdown-menu": "1.4.0",
     "@atlaskit/field-text": "2.7.0",
-    "@atlaskit/icon": "6.0.0",
-    "@atlaskit/modal-dialog": "1.2.4",
-    "@atlaskit/tabs": "1.2.5",
+    "@atlaskit/icon": "7.0.0",
```

Comment:
Looks like the problem is the icons now inline styles onto the SVG instead of setting a class. In the inline styles width is set to 100% (which is odd because they also set a width property). This 100% width isn't playing well in Safari.